modules/jaggery-apps/user-dashboard/dashboard/samlsso.jag

```diff
@@ -18,10 +19,13 @@ else{
 
 
 if(request.getParameter("login") === 'true'){
+    authzFailure =request.getParameter("authz_failure");
```

Comment:
We have used this variable below  <input type="hidden" name="authz_failure" value="<%=authzFailure%>"/>